flow-decomposition/src/test/java/com/powsybl/flow_decomposition/LossesCompensationTests.java

```diff
@@ -278,4 +278,113 @@ void testLossCompensationComputerCanBeAppliedToMultipleNetworks() {
             assertEquals(losses, load.getP0());
         });
     }
+
+    /*
+        Test flow decomposition with losses compensation in a simple network
+          FR : bus1
+          BE : bus2, bus3
+                                   ____ bus3
+                                  |      |
+                                 l32     d3 (1)
+                                  |
+                bus1 --- l21 --- bus2
+                 |                |
+                g1 (3)           d2 (2)
+
+         Line l32 is open at side 1 (bus 3)
+         Line characteristics: (r,x,g,b) = (0.01, 0.1, 0.0, 0.5)
+     */
+    @Test
+    void testLossCompensationWithLineConnectedToOnlyOneSide() {
```

Comment:
Thank you for this great test !
Could you add a test using only the LossCompensator and not the full flow decomposition please ? 
For example:
```suggestion
    /*
        Test flow decomposition with losses compensation in a simple network
          FR : bus1
          BE : bus2, bus3
                                   ____ bus3
                                  |      |
                                 l32     d3 (1)
                                  |
                bus1 --- l21 --- bus2
                 |                |
                g1 (3)           d2 (2)
         Line l32 is open at side 1 (bus 3)
         Line characteristics: (r,x,g,b) = (0.01, 0.1, 0.0, 0.5)
     */
   @Test
    void toBeRefactoredTest {
        String networkFileName = "lossesCompensatorLineConnectedToOneSide.xiidm";
        Network network = importNetwork(networkFileName);
        assertEquals(2, network.getLoadStream().count());
        LoadFlow.run(network, new LoadFlowParameters().setDc(AC_LOAD_FLOW));
        LossesCompensator lossesCompensator = new LossesCompensator(FlowDecompositionParameters.DEFAULT_LOSSES_COMPENSATION_EPSILON);
        lossesCompensator.run(network);
        assertEquals(5, network.getLoadStream().count());
        Load lossesb1 = network.getLoad("LOSSES b1");
        assertNotNull(lossesb1);
        assertEquals("b1_vl", lossesb1.getTerminal().getVoltageLevel().getId());
        assertEquals(0.061171, lossesb1.getP0(), 1e-6);
        Load lossesb2 = network.getLoad("LOSSES b2");
        assertNotNull(lossesb2);
        assertEquals("b2_vl", lossesb2.getTerminal().getVoltageLevel().getId());
        assertEquals(0.003581, lossesb2.getP0(), 1e-6);
        Load lossesb3 = network.getLoad("LOSSES b3");
        assertNotNull(lossesb3);
        assertEquals("b3_vl", lossesb3.getTerminal().getVoltageLevel().getId());
        assertEquals(0.0, lossesb3.getP0(), 1e-6);
    }
```